src/DataTypes/DataTypeNullable.cpp

```diff
@@ -94,6 +94,27 @@ void DataTypeNullable::forEachChild(const ChildCallback & callback) const
 }
 
 
+std::unique_ptr<ISerialization::SubstreamData> DataTypeNullable::getDynamicSubcolumnData(std::string_view subcolumn_name, const DB::IDataType::SubstreamData & data, bool throw_if_null) const
+{
+    auto nested_type = assert_cast<const DataTypeNullable &>(*data.type).nested_data_type;
+    auto nested_data = std::make_unique<ISerialization::SubstreamData>(nested_type->getDefaultSerialization());
```

Comment:
It seems you don't need to use `std::make_unique` here because you don't pass object ownership anywhere, and it's destroyed at the end of the function.